flashinfer/gemm/kernels/tgv_gemm_cute_ext.py

```
@@ -0,0 +1,1336 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+"""
+TGV GEMM (low-latency Blackwell GEMM) — CuTe Ext (``cute_ext``)
+implementation. Companion to ``include/flashinfer/gemm/tgv_gemm.cuh``
+(C++).
+
+Toggled by ``use_2cta`` in the constructor:
+  use_2cta=False (default) → 1×1 cluster, 1-CTA tcgen05.mma
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)